### DISCUSSION TITLE
Allow values of hidden column(s) to be used in data links

### DISCUSSION THREAD

**Discussion by Grrrolf** [grafana/grafana#37185] [Dashboards, Panels & Visualizations]
At work we use data links in many of our dashboards. For us it is important that our stake holders can effortlessly use data links. Data links are mainly used to 'click through' to other dashboards for more detailed information and analytics about a product, an account or any other analytics we provide. This works great! 

However,  when I hide (via transform / organize fields in Grafana 8.0..6) the column(s) we do not want to show the data link does not use the hidden column's value to render the URL. As a result we need to show columns in the source table panel which have no value for the source table panel but are only used to enable a seamless 'click through' user experience.

To keep the source table panel as focussed as possible on answering questions, we would like to hide the columns which are only used to provide the target dashboard with the data needed for filling the template variables. This helps our stake holders focus on the important things.

_**An example (using unrelated data):**_

I have the following table with data:
![image](https://user-images.githubusercontent.com/10052794/126882880-6a4ac609-2abe-492f-ab07-6a89732c2643.png)
When I hover over value in the 'release_name' column I will see in the bottom of my browser the URL Grafana renders the following URL:
![image](https://user-images.githubusercontent.com/10052794/126882891-ce5d2d52-174b-46ce-9e30-8555198d4dfb.png)
However, when I hide the release_id column:
![image](https://user-images.githubusercontent.com/10052794/126882926-cfcd2215-c6b6-481b-bad6-d72a06a01621.png)
the generated URL looks like this:
![image](https://user-images.githubusercontent.com/10052794/126882903-6b2f584b-1bd2-4a9a-9bf6-f5ac8d968f1c.png)
In other words this URL will not work.

Please consider allowing values of hidden columns to be used in data links

_My apologies for using unprofessional data in the examples._

**Comment by torkelo** [comment]
Duplicate of https://github.com/grafana/grafana/issues/24092

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
